app/redux/Counter/counterSlice.ts

```diff
@@ -0,0 +1,20 @@
+import {createSlice} from '@reduxjs/toolkit';
+
+export const initialState = {value: 0};
+
+export const counterSlice = createSlice({
+  name: 'counter',
+  initialState,
+  reducers: {
+    increment: (state, action) => {
+      state.value += Number(action.payload);
```

Comment:
no need to cast number

Reply:
need to cast it to Number because the typeof action?.payload is string